exporters/otlp/src/otlp_environment.cc

```diff
@@ -1125,6 +1157,174 @@ std::string GetOtlpDefaultLogsCompression()
   return std::string{"none"};
 }
 
+std::uint32_t GetOtlpDefaultTracesRetryMaxAttempts()
+{
+  constexpr char kSignalEnv[]  = "OTEL_EXPORTER_OTLP_TRACES_RETRY_MAX_ATTEMPTS";
+  constexpr char kGenericEnv[] = "OTEL_EXPORTER_OTLP_RETRY_MAX_ATTEMPTS";
+  std::uint32_t value{};
+
+  if (GetUintDualEnvVar(kSignalEnv, kGenericEnv, value))
+  {
+    return value;
+  }
+
+  return 5U;
+}
+
+std::uint32_t GetOtlpDefaultMetricsRetryMaxAttempts()
+{
+  constexpr char kSignalEnv[]  = "OTEL_EXPORTER_OTLP_METRICS_RETRY_MAX_ATTEMPTS";
+  constexpr char kGenericEnv[] = "OTEL_EXPORTER_OTLP_RETRY_MAX_ATTEMPTS";
```

Comment:
These new environment variables are not in the spec (yet).

Please use the OTEL_CPP_xxx namespace, as in `OTEL_CPP_EXPORTER_OTLP_METRICS_RETRY_MAX_ATTEMPTS`, and likewise for all friends.

Once the spec is extended, this will be revisited to use the official names.